2019-12-29 17:58:57,455 ERROR 15136 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:201:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 15136
hostname: LAPTOP-FKOQDFD9

2019-12-29 19:13:05,096 ERROR 15136 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:201:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 15136
hostname: LAPTOP-FKOQDFD9

2019-12-29 21:04:03,665 ERROR 15136 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:201:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 15136
hostname: LAPTOP-FKOQDFD9

2019-12-30 10:45:31,407 ERROR 4796 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:201:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 4796
hostname: LAPTOP-FKOQDFD9

2019-12-31 11:50:06,982 ERROR 32016 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:201:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 32016
hostname: LAPTOP-FKOQDFD9

2019-12-31 16:12:49,894 ERROR 32016 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:201:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 32016
hostname: LAPTOP-FKOQDFD9

2020-01-02 10:44:47,334 ERROR 18376 [-/127.0.0.1/-/4ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 10:44:50,363 ERROR 18376 [-/127.0.0.1/-/1ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{}" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{}\""
kind: "string"
value: {}
path: "cid"
reason: null
message: "Cast to string failed for value \"{}\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 10:44:51,809 ERROR 18376 [-/127.0.0.1/-/2ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 10:44:59,718 ERROR 18376 [-/127.0.0.1/-/1ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{}" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{}\""
kind: "string"
value: {}
path: "cid"
reason: null
message: "Cast to string failed for value \"{}\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 10:45:02,071 ERROR 18376 [-/127.0.0.1/-/1ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 10:47:28,654 ERROR 18376 [-/127.0.0.1/-/1ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 10:47:37,753 ERROR 18376 [-/127.0.0.1/-/1ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{}" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{}\""
kind: "string"
value: {}
path: "cid"
reason: null
message: "Cast to string failed for value \"{}\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 10:47:54,471 ERROR 18376 [-/127.0.0.1/-/1ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 11:00:50,719 ERROR 18376 [-/127.0.0.1/-/1ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 11:00:52,871 ERROR 18376 [-/127.0.0.1/-/1ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 11:00:53,094 ERROR 18376 [-/127.0.0.1/-/2ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 11:00:53,332 ERROR 18376 [-/127.0.0.1/-/1ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 11:00:53,871 ERROR 18376 [-/127.0.0.1/-/1ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 11:00:54,089 ERROR 18376 [-/127.0.0.1/-/2ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 11:00:54,123 ERROR 18376 [-/127.0.0.1/-/1ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-02 11:00:54,325 ERROR 18376 [-/127.0.0.1/-/2ms POST /v1/isCollection] nodejs.CastError: Cast to string failed for value "{ id: '238bc2e023844769a6b67d9a4c04b2ea' }" at path "cid" for model "Collection"
    at new CastError (D:\前端\project\mall\server\node_modules\mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:4698:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:1876:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\mongoose\lib\query.js:2131:8)
    at model.Query._wrappedThunk [as _findOne] (D:\前端\project\mall\server\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\""
kind: "string"
value: {"id":"238bc2e023844769a6b67d9a4c04b2ea"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '238bc2e023844769a6b67d9a4c04b2ea' }\" at path \"cid\" for model \"Collection\""
name: "CastError"
model: undefined
pid: 18376
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:37:43,397 ERROR 6648 [-/127.0.0.1/-/201ms POST /v1/editCart] nodejs.CastError: Cast to number failed for value "66074cd2d8464dcc9b2d7fbef5b417d9" at path "count"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaNumber.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:361:11)
    at SchemaNumber.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaNumber.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaNumber.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:413:14)
    at SchemaNumber.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at castUpdateVal (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:512:19)
    at walkUpdatePath (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:339:22)
    at castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:96:18)
    at model.Query._castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4480:10)
    at castDoc (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4508:18)
    at model.Query.Query._findAndModify (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3471:22)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3021:8)
    at model.Query._wrappedThunk [as _findOneAndUpdate] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"66074cd2d8464dcc9b2d7fbef5b417d9\""
kind: "number"
value: "66074cd2d8464dcc9b2d7fbef5b417d9"
path: "count"
reason: {"generatedMessage":true,"code":"ERR_ASSERTION","actual":false,"expected":true,"operator":"=="}
message: "Cast to number failed for value \"66074cd2d8464dcc9b2d7fbef5b417d9\" at path \"count\""
name: "CastError"
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:38:01,130 ERROR 6648 [-/127.0.0.1/-/3ms POST /v1/editCart] nodejs.CastError: Cast to number failed for value "4cf7744443f94557b7c6ad37dca9c4db" at path "count"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaNumber.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:361:11)
    at SchemaNumber.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaNumber.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaNumber.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:413:14)
    at SchemaNumber.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at castUpdateVal (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:512:19)
    at walkUpdatePath (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:339:22)
    at castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:96:18)
    at model.Query._castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4480:10)
    at castDoc (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4508:18)
    at model.Query.Query._findAndModify (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3471:22)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3021:8)
    at model.Query._wrappedThunk [as _findOneAndUpdate] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"4cf7744443f94557b7c6ad37dca9c4db\""
kind: "number"
value: "4cf7744443f94557b7c6ad37dca9c4db"
path: "count"
reason: {"generatedMessage":true,"code":"ERR_ASSERTION","actual":false,"expected":true,"operator":"=="}
message: "Cast to number failed for value \"4cf7744443f94557b7c6ad37dca9c4db\" at path \"count\""
name: "CastError"
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:38:01,939 ERROR 6648 [-/127.0.0.1/-/2ms POST /v1/editCart] nodejs.CastError: Cast to number failed for value "4cf7744443f94557b7c6ad37dca9c4db" at path "count"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaNumber.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:361:11)
    at SchemaNumber.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaNumber.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaNumber.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:413:14)
    at SchemaNumber.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at castUpdateVal (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:512:19)
    at walkUpdatePath (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:339:22)
    at castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:96:18)
    at model.Query._castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4480:10)
    at castDoc (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4508:18)
    at model.Query.Query._findAndModify (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3471:22)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3021:8)
    at model.Query._wrappedThunk [as _findOneAndUpdate] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"4cf7744443f94557b7c6ad37dca9c4db\""
kind: "number"
value: "4cf7744443f94557b7c6ad37dca9c4db"
path: "count"
reason: {"generatedMessage":true,"code":"ERR_ASSERTION","actual":false,"expected":true,"operator":"=="}
message: "Cast to number failed for value \"4cf7744443f94557b7c6ad37dca9c4db\" at path \"count\""
name: "CastError"
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:38:57,436 ERROR 6648 [-/127.0.0.1/-/1ms POST /v1/editCart] nodejs.CastError: Cast to number failed for value "66074cd2d8464dcc9b2d7fbef5b417d9" at path "count"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaNumber.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:361:11)
    at SchemaNumber.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaNumber.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaNumber.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:413:14)
    at SchemaNumber.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at castUpdateVal (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:512:19)
    at walkUpdatePath (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:339:22)
    at castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:96:18)
    at model.Query._castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4480:10)
    at castDoc (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4508:18)
    at model.Query.Query._findAndModify (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3471:22)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3021:8)
    at model.Query._wrappedThunk [as _findOneAndUpdate] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"66074cd2d8464dcc9b2d7fbef5b417d9\""
kind: "number"
value: "66074cd2d8464dcc9b2d7fbef5b417d9"
path: "count"
reason: {"generatedMessage":true,"code":"ERR_ASSERTION","actual":false,"expected":true,"operator":"=="}
message: "Cast to number failed for value \"66074cd2d8464dcc9b2d7fbef5b417d9\" at path \"count\""
name: "CastError"
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:39:38,023 ERROR 6648 [-/127.0.0.1/-/1ms POST /v1/editCart] nodejs.CastError: Cast to number failed for value "66074cd2d8464dcc9b2d7fbef5b417d9" at path "count"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaNumber.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:361:11)
    at SchemaNumber.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaNumber.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaNumber.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:413:14)
    at SchemaNumber.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at castUpdateVal (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:512:19)
    at walkUpdatePath (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:339:22)
    at castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:96:18)
    at model.Query._castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4480:10)
    at castDoc (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4508:18)
    at model.Query.Query._findAndModify (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3471:22)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3021:8)
    at model.Query._wrappedThunk [as _findOneAndUpdate] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"66074cd2d8464dcc9b2d7fbef5b417d9\""
kind: "number"
value: "66074cd2d8464dcc9b2d7fbef5b417d9"
path: "count"
reason: {"generatedMessage":true,"code":"ERR_ASSERTION","actual":false,"expected":true,"operator":"=="}
message: "Cast to number failed for value \"66074cd2d8464dcc9b2d7fbef5b417d9\" at path \"count\""
name: "CastError"
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:40:08,400 ERROR 6648 [-/127.0.0.1/-/1ms POST /v1/editCart] nodejs.CastError: Cast to number failed for value "66074cd2d8464dcc9b2d7fbef5b417d9" at path "count"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaNumber.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:361:11)
    at SchemaNumber.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaNumber.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaNumber.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:413:14)
    at SchemaNumber.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at castUpdateVal (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:512:19)
    at walkUpdatePath (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:339:22)
    at castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:96:18)
    at model.Query._castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4480:10)
    at castDoc (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4508:18)
    at model.Query.Query._findAndModify (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3471:22)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3021:8)
    at model.Query._wrappedThunk [as _findOneAndUpdate] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"66074cd2d8464dcc9b2d7fbef5b417d9\""
kind: "number"
value: "66074cd2d8464dcc9b2d7fbef5b417d9"
path: "count"
reason: {"generatedMessage":true,"code":"ERR_ASSERTION","actual":false,"expected":true,"operator":"=="}
message: "Cast to number failed for value \"66074cd2d8464dcc9b2d7fbef5b417d9\" at path \"count\""
name: "CastError"
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:40:09,494 ERROR 6648 [-/127.0.0.1/-/1ms POST /v1/editCart] nodejs.CastError: Cast to number failed for value "66074cd2d8464dcc9b2d7fbef5b417d9" at path "count"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaNumber.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:361:11)
    at SchemaNumber.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaNumber.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaNumber.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:413:14)
    at SchemaNumber.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at castUpdateVal (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:512:19)
    at walkUpdatePath (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:339:22)
    at castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:96:18)
    at model.Query._castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4480:10)
    at castDoc (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4508:18)
    at model.Query.Query._findAndModify (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3471:22)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3021:8)
    at model.Query._wrappedThunk [as _findOneAndUpdate] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"66074cd2d8464dcc9b2d7fbef5b417d9\""
kind: "number"
value: "66074cd2d8464dcc9b2d7fbef5b417d9"
path: "count"
reason: {"generatedMessage":true,"code":"ERR_ASSERTION","actual":false,"expected":true,"operator":"=="}
message: "Cast to number failed for value \"66074cd2d8464dcc9b2d7fbef5b417d9\" at path \"count\""
name: "CastError"
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:40:10,458 ERROR 6648 [-/127.0.0.1/-/1ms POST /v1/editCart] nodejs.CastError: Cast to number failed for value "4cf7744443f94557b7c6ad37dca9c4db" at path "count"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaNumber.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:361:11)
    at SchemaNumber.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaNumber.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaNumber.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:413:14)
    at SchemaNumber.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at castUpdateVal (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:512:19)
    at walkUpdatePath (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:339:22)
    at castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:96:18)
    at model.Query._castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4480:10)
    at castDoc (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4508:18)
    at model.Query.Query._findAndModify (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3471:22)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3021:8)
    at model.Query._wrappedThunk [as _findOneAndUpdate] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"4cf7744443f94557b7c6ad37dca9c4db\""
kind: "number"
value: "4cf7744443f94557b7c6ad37dca9c4db"
path: "count"
reason: {"generatedMessage":true,"code":"ERR_ASSERTION","actual":false,"expected":true,"operator":"=="}
message: "Cast to number failed for value \"4cf7744443f94557b7c6ad37dca9c4db\" at path \"count\""
name: "CastError"
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:40:11,684 ERROR 6648 [-/127.0.0.1/-/2ms POST /v1/editCart] nodejs.CastError: Cast to number failed for value "418fc60784d04e71beffe1ce5174c947" at path "count"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaNumber.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:361:11)
    at SchemaNumber.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaNumber.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaNumber.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:413:14)
    at SchemaNumber.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at castUpdateVal (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:512:19)
    at walkUpdatePath (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:339:22)
    at castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:96:18)
    at model.Query._castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4480:10)
    at castDoc (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4508:18)
    at model.Query.Query._findAndModify (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3471:22)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3021:8)
    at model.Query._wrappedThunk [as _findOneAndUpdate] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"418fc60784d04e71beffe1ce5174c947\""
kind: "number"
value: "418fc60784d04e71beffe1ce5174c947"
path: "count"
reason: {"generatedMessage":true,"code":"ERR_ASSERTION","actual":false,"expected":true,"operator":"=="}
message: "Cast to number failed for value \"418fc60784d04e71beffe1ce5174c947\" at path \"count\""
name: "CastError"
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:40:25,406 ERROR 6648 [-/127.0.0.1/-/2ms POST /v1/editCart] nodejs.CastError: Cast to number failed for value "66074cd2d8464dcc9b2d7fbef5b417d9" at path "count"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaNumber.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:361:11)
    at SchemaNumber.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaNumber.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaNumber.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\number.js:413:14)
    at SchemaNumber.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at castUpdateVal (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:512:19)
    at walkUpdatePath (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:339:22)
    at castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\castUpdate.js:96:18)
    at model.Query._castUpdate (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4480:10)
    at castDoc (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4508:18)
    at model.Query.Query._findAndModify (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3471:22)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:3021:8)
    at model.Query._wrappedThunk [as _findOneAndUpdate] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"66074cd2d8464dcc9b2d7fbef5b417d9\""
kind: "number"
value: "66074cd2d8464dcc9b2d7fbef5b417d9"
path: "count"
reason: {"generatedMessage":true,"code":"ERR_ASSERTION","actual":false,"expected":true,"operator":"=="}
message: "Cast to number failed for value \"66074cd2d8464dcc9b2d7fbef5b417d9\" at path \"count\""
name: "CastError"
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:47:15,608 ERROR 6648 [-/127.0.0.1/-/5ms POST /v1/deleteShop] nodejs.CastError: Cast to string failed for value "{ '66074cd2d8464dcc9b2d7fbef5b417d9': '' }" at path "cid" for model "ShopList"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4683:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:1861:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:2811:8)
    at model.Query._wrappedThunk [as _deleteMany] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ '66074cd2d8464dcc9b2d7fbef5b417d9': '' }\""
kind: "string"
value: {"66074cd2d8464dcc9b2d7fbef5b417d9":""}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ '66074cd2d8464dcc9b2d7fbef5b417d9': '' }\" at path \"cid\" for model \"ShopList\""
name: "CastError"
model: undefined
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:50:42,782 ERROR 6648 [-/127.0.0.1/-/2ms POST /v1/deleteShop] nodejs.CastError: Cast to string failed for value "{ '66074cd2d8464dcc9b2d7fbef5b417d9': '' }" at path "cid" for model "ShopList"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4683:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:1861:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:2811:8)
    at model.Query._wrappedThunk [as _deleteMany] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ '66074cd2d8464dcc9b2d7fbef5b417d9': '' }\""
kind: "string"
value: {"66074cd2d8464dcc9b2d7fbef5b417d9":""}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ '66074cd2d8464dcc9b2d7fbef5b417d9': '' }\" at path \"cid\" for model \"ShopList\""
name: "CastError"
model: undefined
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:50:46,400 ERROR 6648 [-/127.0.0.1/-/1ms POST /v1/deleteShop] nodejs.CastError: Cast to string failed for value "{ '66074cd2d8464dcc9b2d7fbef5b417d9': '' }" at path "cid" for model "ShopList"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4683:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:1861:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:2811:8)
    at model.Query._wrappedThunk [as _deleteMany] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ '66074cd2d8464dcc9b2d7fbef5b417d9': '' }\""
kind: "string"
value: {"66074cd2d8464dcc9b2d7fbef5b417d9":""}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ '66074cd2d8464dcc9b2d7fbef5b417d9': '' }\" at path \"cid\" for model \"ShopList\""
name: "CastError"
model: undefined
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:50:46,406 ERROR 6648 [-/127.0.0.1/-/1ms POST /v1/deleteShop] nodejs.CastError: Cast to string failed for value "{ '4cf7744443f94557b7c6ad37dca9c4db': '' }" at path "cid" for model "ShopList"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4683:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:1861:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:2811:8)
    at model.Query._wrappedThunk [as _deleteMany] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ '4cf7744443f94557b7c6ad37dca9c4db': '' }\""
kind: "string"
value: {"4cf7744443f94557b7c6ad37dca9c4db":""}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ '4cf7744443f94557b7c6ad37dca9c4db': '' }\" at path \"cid\" for model \"ShopList\""
name: "CastError"
model: undefined
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:50:58,826 ERROR 6648 [-/127.0.0.1/-/1ms POST /v1/deleteShop] nodejs.CastError: Cast to string failed for value "{ '66074cd2d8464dcc9b2d7fbef5b417d9': '' }" at path "cid" for model "ShopList"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4683:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:1861:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:2811:8)
    at model.Query._wrappedThunk [as _deleteMany] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ '66074cd2d8464dcc9b2d7fbef5b417d9': '' }\""
kind: "string"
value: {"66074cd2d8464dcc9b2d7fbef5b417d9":""}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ '66074cd2d8464dcc9b2d7fbef5b417d9': '' }\" at path \"cid\" for model \"ShopList\""
name: "CastError"
model: undefined
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-05 15:52:10,803 ERROR 6648 [-/127.0.0.1/-/1ms POST /v1/deleteShop] nodejs.CastError: Cast to string failed for value "{ id: '66074cd2d8464dcc9b2d7fbef5b417d9' }" at path "cid" for model "ShopList"
    at new CastError (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\error\cast.js:39:11)
    at SchemaString.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\string.js:583:11)
    at SchemaString.SchemaType.applySetters (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:975:12)
    at SchemaString.SchemaType._castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1389:15)
    at SchemaString.castForQuery (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schema\string.js:645:15)
    at SchemaString.SchemaType.castForQueryWrapper (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\schematype.js:1358:15)
    at cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\cast.js:260:34)
    at model.Query.Query.cast (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:4683:12)
    at model.Query.Query._castConditions (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:1861:10)
    at model.Query.<anonymous> (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\query.js:2811:8)
    at model.Query._wrappedThunk [as _deleteMany] (D:\前端\project\mall\server\node_modules\_mongoose@5.8.4@mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at D:\前端\project\mall\server\node_modules\_kareem@2.3.1@kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
stringValue: "\"{ id: '66074cd2d8464dcc9b2d7fbef5b417d9' }\""
kind: "string"
value: {"id":"66074cd2d8464dcc9b2d7fbef5b417d9"}
path: "cid"
reason: null
message: "Cast to string failed for value \"{ id: '66074cd2d8464dcc9b2d7fbef5b417d9' }\" at path \"cid\" for model \"ShopList\""
name: "CastError"
model: undefined
pid: 6648
hostname: LAPTOP-FKOQDFD9

2020-01-06 14:35:11,966 ERROR 11872 [-/127.0.0.1/-/9ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 14:36:18,100 ERROR 11872 [-/127.0.0.1/-/3ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 14:36:50,988 ERROR 11872 [-/127.0.0.1/-/3ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 14:37:03,949 ERROR 11872 [-/127.0.0.1/-/3ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 14:37:44,755 ERROR 11872 [-/127.0.0.1/-/2ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 14:38:37,553 ERROR 11872 [-/127.0.0.1/-/3ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 14:39:54,772 ERROR 11872 [-/127.0.0.1/-/2ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 14:40:16,117 ERROR 11872 [-/127.0.0.1/-/3ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 14:40:24,675 ERROR 11872 [-/127.0.0.1/-/2ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 14:41:56,237 ERROR 11872 [-/127.0.0.1/-/2ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 15:47:30,452 ERROR 11872 [-/127.0.0.1/-/4ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 15:47:30,699 ERROR 11872 [-/127.0.0.1/-/2ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 15:47:40,073 ERROR 11872 [-/127.0.0.1/-/2ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 15:49:05,194 ERROR 11872 [-/127.0.0.1/-/3ms POST /v1/order] nodejs.TypeError: Cannot read property 'count' of undefined
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:197:34)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 16:53:39,005 ERROR 11872 [-/127.0.0.1/-/4ms POST /v1/order] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:184:40)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 16:54:43,763 ERROR 11872 [-/127.0.0.1/-/4ms POST /v1/order] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:184:40)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 16:55:42,585 ERROR 11872 [-/127.0.0.1/-/2ms POST /v1/order] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:184:40)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 16:58:17,190 ERROR 11872 [-/127.0.0.1/-/1ms POST /v1/order] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:184:40)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 16:58:31,706 ERROR 11872 [-/127.0.0.1/-/3ms POST /v1/order] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:184:40)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 16:58:43,582 ERROR 11872 [-/127.0.0.1/-/2ms POST /v1/order] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:184:40)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 16:59:19,204 ERROR 11872 [-/127.0.0.1/-/3ms POST /v1/order] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:184:40)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 17:00:12,881 ERROR 11872 [-/127.0.0.1/-/3ms POST /v1/order] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.order (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:184:40)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-06 20:24:25,490 ERROR 11872 [-/127.0.0.1/-/10ms POST /v1/saveUser] nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\1578313465481+7__algf6448+l105e0888e60d944f200c17c574.png'
ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\1578313465481+7__algf6448+l105e0888e60d944f200c17c574.png'
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\前端\\project\\mall\\server\\app\\public\\avatar\\1578313465481+7__algf6448+l105e0888e60d944f200c17c574.png"
name: "ENOENTError"
pid: 11872
hostname: LAPTOP-FKOQDFD9

2020-01-07 15:32:21,234 ERROR 5316 [-/127.0.0.1/-/35ms POST /v1/goodsOne/comment] nodejs.TypeError: _base64.includes is not a function
    at ToolsService.getUploadFile (D:\前端\project\mall\server\app\service\tools.js:28:21)
    at OperatingGoodsController.comment (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:249:52)
    at Object.callFn (D:\前端\project\mall\server\node_modules\_egg-core@4.17.4@egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\前端\project\mall\server\node_modules\_egg-core@4.17.4@egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\前端\project\mall\server\node_modules\_@eggjs_router@2.0.0@@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\前端\project\mall\server\node_modules\_@eggjs_router@2.0.0@@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\前端\project\mall\server\node_modules\_koa-compose@3.2.1@koa-compose\index.js:44:32)
    at next (D:\前端\project\mall\server\node_modules\_koa-compose@3.2.1@koa-compose\index.js:45:18)
    at D:\前端\project\mall\server\node_modules\_@eggjs_router@2.0.0@@eggjs\router\lib\router.js:190:18
    at dispatch (D:\前端\project\mall\server\node_modules\_koa-compose@3.2.1@koa-compose\index.js:44:32)
    at D:\前端\project\mall\server\node_modules\_koa-compose@3.2.1@koa-compose\index.js:36:12
    at dispatch (D:\前端\project\mall\server\node_modules\_@eggjs_router@2.0.0@@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\前端\project\mall\server\node_modules\_koa-compose@4.1.0@koa-compose\index.js:42:32)
    at D:\前端\project\mall\server\app\middleware\isLogin.js:15:19
    at dispatch (D:\前端\project\mall\server\node_modules\_koa-compose@4.1.0@koa-compose\index.js:42:32)
    at D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:56

pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-07 15:33:02,716 ERROR 5316 [-/127.0.0.1/-/30ms POST /v1/goodsOne/comment] nodejs.TypeError: _base64.includes is not a function
    at ToolsService.getUploadFile (D:\前端\project\mall\server\app\service\tools.js:28:21)
    at OperatingGoodsController.comment (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:249:52)
    at Object.callFn (D:\前端\project\mall\server\node_modules\_egg-core@4.17.4@egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\前端\project\mall\server\node_modules\_egg-core@4.17.4@egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\前端\project\mall\server\node_modules\_@eggjs_router@2.0.0@@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\前端\project\mall\server\node_modules\_@eggjs_router@2.0.0@@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\前端\project\mall\server\node_modules\_koa-compose@3.2.1@koa-compose\index.js:44:32)
    at next (D:\前端\project\mall\server\node_modules\_koa-compose@3.2.1@koa-compose\index.js:45:18)
    at D:\前端\project\mall\server\node_modules\_@eggjs_router@2.0.0@@eggjs\router\lib\router.js:190:18
    at dispatch (D:\前端\project\mall\server\node_modules\_koa-compose@3.2.1@koa-compose\index.js:44:32)
    at D:\前端\project\mall\server\node_modules\_koa-compose@3.2.1@koa-compose\index.js:36:12
    at dispatch (D:\前端\project\mall\server\node_modules\_@eggjs_router@2.0.0@@eggjs\router\lib\router.js:195:33)
    at dispatch (D:\前端\project\mall\server\node_modules\_koa-compose@4.1.0@koa-compose\index.js:42:32)
    at D:\前端\project\mall\server\app\middleware\isLogin.js:15:19
    at dispatch (D:\前端\project\mall\server\node_modules\_koa-compose@4.1.0@koa-compose\index.js:42:32)
    at D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:56

pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-07 15:37:04,018 ERROR 5316 [-/127.0.0.1/-/26ms POST /v1/goodsOne/comment] nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\15783826240149m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png'
ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\15783826240149m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png'
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\前端\\project\\mall\\server\\app\\public\\avatar\\15783826240149m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png"
name: "ENOENTError"
pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-07 15:37:28,226 ERROR 5316 [-/127.0.0.1/-/32ms POST /v1/goodsOne/comment] nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\15783826482259m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png'
ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\15783826482259m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png'
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\前端\\project\\mall\\server\\app\\public\\avatar\\15783826482259m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png"
name: "ENOENTError"
pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-07 15:37:30,563 ERROR 5316 [-/127.0.0.1/-/31ms POST /v1/goodsOne/comment] nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\15783826505629m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png'
ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\15783826505629m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png'
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\前端\\project\\mall\\server\\app\\public\\avatar\\15783826505629m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png"
name: "ENOENTError"
pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-07 15:38:13,351 ERROR 5316 [-/127.0.0.1/-/28ms POST /v1/goodsOne/comment] nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\15783826933509m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png'
ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\15783826933509m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png'
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\前端\\project\\mall\\server\\app\\public\\avatar\\15783826933509m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png"
name: "ENOENTError"
pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-07 15:38:16,763 ERROR 5316 [-/127.0.0.1/-/22ms POST /v1/goodsOne/comment] nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\15783826967629m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png'
ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\15783826967629m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png'
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\前端\\project\\mall\\server\\app\\public\\avatar\\15783826967629m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png"
name: "ENOENTError"
pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-07 15:38:32,360 ERROR 5316 [-/127.0.0.1/-/23ms POST /v1/goodsOne/comment] nodejs.ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\15783827123599m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png'
ENOENTError: ENOENT: no such file or directory, open 'D:\前端\project\mall\server\app\public\avatar\15783827123599m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png'
errno: -4058
code: "ENOENT"
syscall: "open"
path: "D:\\前端\\project\\mall\\server\\app\\public\\avatar\\15783827123599m_10_69de_a-i+f5e1332cbd77ce02e609bb934.png"
name: "ENOENTError"
pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-07 18:01:08,757 ERROR 5316 [-/127.0.0.1/-/5ms POST /v1/addShop] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.addShop (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:61:38)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-07 18:01:11,071 ERROR 5316 [-/127.0.0.1/-/4ms POST /v1/addShop] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.addShop (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:61:38)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-07 18:01:11,389 ERROR 5316 [-/127.0.0.1/-/3ms POST /v1/addShop] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.addShop (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:61:38)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-07 18:01:11,927 ERROR 5316 [-/127.0.0.1/-/5ms POST /v1/addShop] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.addShop (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:61:38)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-07 18:01:15,045 ERROR 5316 [-/127.0.0.1/-/3ms POST /v1/addShop] nodejs.TypeError: Cannot read property 'present_price' of null
    at OperatingGoodsController.addShop (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:61:38)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 5316
hostname: LAPTOP-FKOQDFD9

2020-01-13 16:14:20,155 ERROR 4064 [-/127.0.0.1/-/4ms POST /v1/collection] nodejs.TypeError: Cannot read property 'id' of null
    at OperatingGoodsController.collection (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:16:24)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 4064
hostname: LAPTOP-FKOQDFD9

2020-01-13 16:14:20,914 ERROR 4064 [-/127.0.0.1/-/3ms POST /v1/collection] nodejs.TypeError: Cannot read property 'id' of null
    at OperatingGoodsController.collection (D:\前端\project\mall\server\app\controller\api\operatingGoods.js:16:24)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async D:\前端\project\mall\server\app\middleware\isLogin.js:15:13
    at async D:\前端\project\mall\server\node_modules\_egg-development@2.4.3@egg-development\app\middleware\egg_loader_trace.js:9:50
    at async xframe (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xframe.js:7:5)
    at async xssProtection (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\xssProtection.js:7:5)
    at async nosniff (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\nosniff.js:8:5)
    at async noopen (D:\前端\project\mall\server\node_modules\_egg-security@2.7.1@egg-security\lib\middlewares\noopen.js:8:5)
    at async session (D:\前端\project\mall\server\node_modules\_koa-session@5.12.3@koa-session\index.js:41:7)
    at async bodyParser (D:\前端\project\mall\server\node_modules\_koa-bodyparser@4.2.1@koa-bodyparser\index.js:86:5)
    at async D:\前端\project\mall\server\node_modules\_koa-static-cache@5.1.2@koa-static-cache\index.js:51:63
    at async notfound (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\notfound.js:5:5)
    at async meta (D:\前端\project\mall\server\node_modules\_egg@2.26.0@egg\app\middleware\meta.js:12:5)

pid: 4064
hostname: LAPTOP-FKOQDFD9

